test/system/trader.js

```diff
@@ -0,0 +1,179 @@
+const cryptoClient = require('../../lib/cryptonite-client');
```

Comment:
Prefer PascalCase (i.e. CryptoniteClient) for class names. 'cryptoClient' implies a function or variable